backend/src/main/java/com/cruru/applicant/service/ApplicantService.java

```diff
@@ -70,4 +71,18 @@ private QnaResponse toQnaResponse(Answer answer) {
         Question question = answer.getQuestion();
         return new QnaResponse(question.getSequence(), question.getContent(), answer.getContent());
     }
+
+    @Transactional
+    public void rejectById(long id) {
```

Comment:
```suggestion
    public void reject(long id) {
```

reject로 작성해도 의미가 전달될 것 같아요.